scripts/set-last-commit.sh

```diff
@@ -6,15 +6,15 @@ if ! git remote | grep -q upstream; then
 fi
 
 # Fetch latest changes from the original repository
-git fetch upstream
+git fetch --unshallow upstream
 
 # Find the last common commit
 VITE_LAST_ORIGINAL_COMMIT=$(git merge-base HEAD upstream/main)
 
 # Check if the command succeeded and VITE_LAST_ORIGINAL_COMMIT is not empty
 if [ -z "$VITE_LAST_ORIGINAL_COMMIT" ]; then
   echo "Error: Unable to determine the last original commit."
-  exit 1
+  exit 0
```

Comment:
i believe if we have an error we should have a nonzero exit code

Reply:
good point, I'll update the message to not say error. I am purposely returning non error code to not break deploy if this script fails. Since in theory not having a common commit history should not break deploys